From 8e82734b27588d7c79720d58c4323edaabfb7bfd Mon Sep 17 00:00:00 2001
From: Pedro Falcato <pedro.falcato@gmail.com>
Date: Sun, 13 Apr 2025 00:27:44 +0000
Subject: [PATCH 3/3] Make newuidmap and newgidmap setuid

We don't support file capabilities, so use good old setuid.

Signed-off-by: Pedro Falcato <pedro.falcato@gmail.com>
---
 shadow-utils.spec | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/shadow-utils.spec b/shadow-utils.spec
index 5feb4f8..fda3d3d 100644
--- a/shadow-utils.spec
+++ b/shadow-utils.spec
@@ -202,8 +202,8 @@ rm -f $RPM_BUILD_ROOT/%{_libdir}/libsubid.a
 %attr(4755,root,root) %{_bindir}/gpasswd
 %{_bindir}/lastlog
 %attr(4755,root,root) %{_bindir}/newgrp
-%attr(0755,root,root) %caps(cap_setgid=ep) %{_bindir}/newgidmap
-%attr(0755,root,root) %caps(cap_setuid=ep) %{_bindir}/newuidmap
+%attr(4755,root,root) %{_bindir}/newgidmap
+%attr(4755,root,root) %{_bindir}/newuidmap
 %attr(4755,root,root) %{_bindir}/passwd
 %{_sbindir}/adduser
 %attr(0755,root,root)   %{_sbindir}/user*
-- 
2.46.2

